### ISSUE TITLE
Cannot delete characters in Input element

### ISSUE THREAD

**Issue by ividyon** [FRACerqueira/PromptPlus#69]
I have the following code:

```cs
                    var input = PromptPlus.Input("Input new delay (in milliseconds)")
                        .AcceptInput(char.IsNumber)
                        .AddValidators(PromptValidators.IsTypeUInt16("Input is not within valid range"))
                        .ValidateOnDemand()
                        .Run();
```

I noticed that I cannot press Backspace, or use the arrow keys to move left/right in this input window. I have to press Esc to cancel out of the whole input and try again, if I make a typo. It's only possible to input characters, not remove characters.

**Comment by FRACerqueira** [comment]
This is a bug and was caused by 'AcceptInput'. I probably need to check the navigation keys before filtering by AcceptInput. I will check.

**Comment by FRACerqueira** [comment]
Fixed scheduled for publication 4.0.6

**Comment by ividyon** [comment]
Very cool.
